src/services/sanity.js

```diff
@@ -325,6 +314,48 @@ export async function fetchUpcomingEvents(brand, { page = 1, limit = 10 } = {})
   return fetchSanity(query, true);
 }
 
+/**
+ * Fetch scheduled releases for a specific brand.
+ *
+ * @param {string} brand - The brand for which to fetch scheduled releasess.
+ * @returns {Promise<Object|null>} - A promise that resolves to an array of scheduled release objects or null if not found.
+ *
+ * @example
+ * fetchScheduledReleases('drumeo', {
+ *   page: 2,
+ *   limit: 20,
+ * })
+ *   .then(content => console.log(content))
+ *   .catch(error => console.error(error));
+ */
+export async function fetchScheduledReleases(brand, { page = 1, limit = 10 }) {
+  const upcomingTypes = getUpcomingEventsTypes();
+  const newTypes = getNewReleasesTypes();
+
+  const scheduledTypes = Object.keys(upcomingTypes).reduce((acc, key) => {
+    acc[key] = [...new Set([...upcomingTypes[key], ...newTypes[key]])];
+    return acc;
+  }, {});
```

Comment:
Proposal, and there may be a reason you didn't do it this way that I didn't understand. But we can simplify the pull from the helper methods by including the brand, and only getting the relevant array back. 

this requires adding a merge method (which I stole from [here](https://stackoverflow.com/questions/1584370/how-to-merge-two-arrays-in-javascript-and-de-duplicate-items)
```
const merge = (a, b, predicate = (a, b) => a === b) => {
    const c = [...a]; // copy to avoid side effects
    // add all items from B to copy C if they're not already present
    b.forEach((bItem) => (c.some((cItem) => predicate(bItem, cItem)) ? null : c.push(bItem)))
    return c;
}
```


```suggestion
  const upcomingTypes = getUpcomingEventsTypes(brand);
  const newTypes = getNewReleasesTypes(brand);

  const scheduledTypes = merge(upcomingTypes, newTypes)
```